app.js

```diff
@@ -38,9 +38,103 @@ function operate(operator, a, b) {
 const display = document.querySelector(".display");
 const btns = document.querySelectorAll("button");
 
-// put number selected on the display
 btns.forEach((button) => {
-  button.addEventListener("click", (e) => {
-    display.innerHTML += button.innerText;
-  });
+  button.addEventListener("click", (e) => calculatorDisplay(button.id));
 });
+
+let firstNum = "";
+let secondNum = "";
+let operator;
+// put number selected on the display
+function calculatorDisplay(value) {
+  function getData(value) {
+    if (
+      display.innerHTML.includes("-") ||
+      display.innerHTML.includes("+") ||
+      display.innerHTML.includes("x") ||
+      display.innerHTML.includes("÷")
```

Comment:
This is brittle, you can replace this with `if (operator)` since you have stored the operator in a variable.